gatsby-node.ts

```diff
@@ -38,7 +38,8 @@ const commonRedirectProps = {
  * @returns boolean for if file is outdated or not
  */
 const checkIsMdxOutdated = (filePath: string): boolean => {
-  const dirname = path.resolve("./")
+  // .replace(/\\./g, "/") to replace \ in windows paths ex: C:\\folder\\myfile.txt becomes C:/folder/myfile.txt
+  const dirname = path.resolve("./").replace(/\\./g, "/")
```

Comment:
It actually should not have the dot.  It is not used for the problem we are trying to solve, and it's deleting the first letter after each slash.
```suggestion
  // .replace(/\\/g, "/") to replace \ in windows paths ex: C:\\folder\\myfile.txt becomes C:/folder/myfile.txt
  const dirname = path.resolve("./").replace(/\\/g, "/")
```

Reply:
Woops, mistyped it. Thanks!